src/test/java/cucumber/steps/StepDefinitions.java

```diff
@@ -0,0 +1,47 @@
+package cucumber.steps;
+
+import api.ApiClient;
+import controllers.DashboardController;
+import controllers.LaunchController;
+import io.cucumber.java.en.When;
+
+import java.util.ArrayList;
+
+import static helpers.UserLogin.getToken;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StepDefinitions {
+    private ApiClient api = new ApiClient();
+    private LaunchController launch = new LaunchController(api);
+    private DashboardController dashboards = new DashboardController(api);
+    private ArrayList<String> listWithDashboardNames;
+
+    @When("user is logged in")
+    public void checkUserIsLoggedIn() {
+        assertNotNull(getToken(), "Authorization failed");
```

Comment:
Whould you add some logging to each step to the log-file readers?

Reply:
add logger to checkUserIsLoggedIn method